sklearn/metrics/_regression.py

```diff
@@ -466,6 +467,9 @@ def mean_squared_log_error(
 
         'uniform_average' :
             Errors of all outputs are averaged with uniform weight.
+            
+        'squared' : bool, default=True
+        If True returns MSLE(mean squared log error) value, if False returns RMSLE(root mean squared log error) value.
```

Comment:
Please add a line break to keep each line below 88 char long.

Reply:
> I have no idea about `black'. How to run on the changed file to fix linting!